sale_view_adjust_oaw/views/sale_order_views.xml

```diff
@@ -1,17 +1,53 @@
 <?xml version="1.0" encoding="utf-8"?>
 <openerp>
-<data>
+    <data>
 
-    <record id="sale_order_form" model="ir.ui.view">
-        <field name="name">sale.order.form</field>
-        <field name="model">sale.order</field>
-        <field name="inherit_id" ref="sale.view_order_form"/>
-        <field name="arch" type="xml">
-            <xpath expr="/form/sheet/notebook/page[1]/field[@name='order_line']/tree[1]/field[@name='product_id'] " position="before">
-                <field name="image_small" widget="image" height="64px"/>
-            </xpath>
-        </field>
-    </record>
+        <record id="sale_margin_sale_order_line_form_ext" model="ir.ui.view">
+            <field name="name">view.order.form.tree.adjust</field>
+            <field name="model">sale.order</field>
+            <field name="inherit_id"
+                   ref="sale_margin.sale_margin_sale_order_line_form"/>
+            <field name="arch" type="xml">
+                <xpath expr="/form/sheet/notebook/page[@string='Order Lines']/field[@name='order_line']/tree[@string='Sales Order Lines']/field[@name='purchase_price']"
```

Comment:
`purchase_price`, `lot_id` and `quant_id` are being removed in this xml and not being added back. If this is the case, we should make them invisible.